src/App.js

```diff
@@ -93,15 +93,29 @@ type State = {
 
 function getAccessibilityFilterFrom(statusString: string): YesNoLimitedUnknown[] {
   const allowedStatuses = yesNoLimitedUnknownArray;
-  if (!statusString) return [].concat(allowedStatuses);
-  const result = statusString.split(/\./).filter(s => includes(allowedStatuses, s));
+
+  // Safe mutable sort as filter always returns a new array.
+  const result = statusString
```

Comment:
MINOR: This could be moved into a small helper function, and used also below.

Reply:
Good catch!